server/lib/xrpl-ws.js

```diff
@@ -0,0 +1,84 @@
+const WebSocket = require('ws');
```

Comment:
This was all here previously - I just removed the `server` and `validations` subscription streams.